src/styles/StyleUtils.js

```diff
@@ -196,14 +196,12 @@ function getZoomCursorStyle(isZoomed, isDragging) {
  * @param {Number} zoomScale
  * @param {Number} containerHeight
  * @param {Number} containerWidth
- * @return {Object}
+ * @param {Boolean} isLoading
+ * @return {Object | undefined}
```

Comment:
According to our [style guide](https://github.com/Expensify/App/blob/main/contributingGuides/STYLE.md#jsdocs): 
> Do not use type unions e.g. {(number|boolean)}.

Reply:
Ok, I think it's acceptable in this case